polaris-react/src/components/Scrollable/Scrollable.tsx

```diff
@@ -166,9 +197,19 @@ function performScrollHint(elem?: HTMLDivElement | null) {
   elem.scrollTo({top: MAX_SCROLL_HINT_DISTANCE, behavior: 'smooth'});
 }
 
-Scrollable.ScrollTo = ScrollTo;
-
-Scrollable.forNode = (node: HTMLElement): HTMLElement | Document => {
+const forNode = (node: HTMLElement): HTMLElement | Document => {
   const closestElement = node.closest(scrollable.selector);
   return closestElement instanceof HTMLElement ? closestElement : document;
 };
+
+type ScrollableType = typeof ScrollableComponent & {
+  ScrollTo: typeof ScrollTo;
+  forNode: typeof forNode;
+};
+
+// @ts-expect-error - expected functions/sub-components are assigned after declaration
+const Scrollable: ScrollableType = ScrollableComponent;
+Scrollable.ScrollTo = ScrollTo;
+Scrollable.forNode = forNode;
```

Comment:
I think these types can be cleaned up a bit. Took a stab at it [here](https://github.com/Shopify/polaris/compare/jennings/scrollable-imperative-handle...scrollable-clean-up-types?expand=1)